openshift-client/pom.xml

```diff
@@ -144,6 +144,12 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
```

Comment:
Is it really necessary ? Would be nice if you could do this without introducing this dependency ;-)

Reply:
@rnetuka : We use jackson.  Could you please update it? I'm planning to do a release today so it would be nice if it could go in.